manifests/upstream/keptn/base/kustomization.yaml

```
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
```

I think there are enough services / deployments in keptn to justify its own namespace
```suggestion
namespace: keptn
```